hedera-node/src/main/java/com/hedera/services/store/models/TokenRelationship.java

```diff
@@ -96,22 +95,25 @@ public void setBalance(long balance) {
 		this.balance = balance;
 	}
 
-	private void adjustBalance(long adjustment) {
+	void adjustBalance(long adjustment) {
 		balanceChange += adjustment;
 		this.balance += adjustment;
 	}
 
 	public void validateAndDissociate(final OptionValidator validator,
```

Comment:
Is this method used?

Reply:
Yes, It's used in the Account.dissociateWith [142-143]